pkg/kubectl/resource/builder.go

```diff
@@ -274,12 +275,48 @@ func (b *Builder) SelectAllParam(selectAll bool) *Builder {
 	return b
 }
 
+func hasNamesArg(args []string) bool {
+	if len(args) > 1 {
+		return true
+	}
+	newArgs := []string{}
+	for _, s := range args {
+		newArgs = append(newArgs, strings.Split(s, ",")...)
+	}
+	args = newArgs
+	for _, s := range args {
+		if !strings.Contains(s, "/") {
+			return false
+		}
+		tuple := strings.Split(s, "/")
+		if len(tuple) != 3 && isGroup(tuple[0]) {
```

Comment:
Ok. Or we can make it a rule to parse it as group/resource in case of ambiguity, which is what you do now.